WcaOnRails/app/models/user.rb

```diff
@@ -155,6 +155,8 @@ def dummy_account?
     wca_id.present? && encrypted_password.blank? && email.casecmp("#{wca_id}@worldcubeassociation.org") == 0
   end
 
+  scope :delegate, -> { where.not(delegate_status: nil) }
```

Comment:
Yeah, there's no obvious way for naming sopes I think. For me `User.delegate.where...` is as good as `User.delegates.where...`, because `User` is singular.